src/mumble/Log.cpp

```diff
@@ -333,35 +333,38 @@ void Log::clearIgnore() {
 	qmIgnore.clear();
 }
 
+/// Estimate the size of buf when encoded as an HTML image tag
+/// with the src attribute set to a base64-encoded data URL, such
+/// as:
+///
+///    <img src="image/jpeg;base64,[base64-of-buf-goes-here]" />
+///
+static unsigned int estimatedDataURLImageSize(const QByteArray &buf) {
+    // We estimate the extra characters (HTML) in the encoded
+    // message to be around 64.
+    const unsigned int htmlExtras = 64U;
```

Comment:
OK, I can live with that compromise -- IMO, most image messages are:
1. One image per message, and
2. No extra content besides that

Maybe add an extra note to the htmlExtras comment clarifying?
"[...] This accounts for the image tag itself, but also a bit of extra HTML from the user."?